fastplotlib/graphics/selectors/_linear_region.py

```diff
@@ -26,16 +26,23 @@ class LinearBoundsFeature(GraphicFeature):
     | "selected_indices" | ``numpy.ndarray`` or ``None`` | selected graphic data indices                                                        |
     | "selected_data"    | ``numpy.ndarray`` or ``None`` | selected graphic data                                                                |
     | "new_data"         | ``(float, float)``            | current bounds in world coordinates, NOT necessarily the same as "selected_indices". |
+    | "move_info"        | ``Move_Info``                 | last position and event source (fill, mesh, or edge line)                            |
+    | "collection_index" | ``int``                       | index of collection                                                                  |
+    | "graphic"          | ``Graphic``                   | the selection graphic                                                                |
+    | "delta"            | ``numpy.ndarray``             | the delta vector of the graphic in NDC                                               |
+    | "world_object"     | ``pygfx.WorldObject``         | pygfx World Object                                                                   |
+    | "index"            | ``None``                      | not used in this feature                                                             |
```

Comment:
let's actually get rid of this for all `Selectors`, I think when I initially was doing this I just adapted from our other `GraphicFeature` classes but I just took a look and there's actually no reason for us to have `index` or `collection_index` within the `pick_info` dict. There's no place in the event handling where it looks for these keys, so we can just remove the clutter :D